test/exception_notifier/email_notifier_test.rb

```diff
@@ -295,30 +297,13 @@ def index; end
       Environment:
       -------------------------------
 
-        * HTTPS                                     : on
```

Comment:
This whole block is very different depending on versions. I'm just simplifying the assertion. We just care that the env is present.